### ISSUE TITLE
MAM03165c sedoheptulose, wrong xref

### ISSUE THREAD

**Issue by ANiknejad** [SysBioChalmers/Human-GEM#310]
Hello,

tricky issue

https://metabolicatlas.org/explore/Fruitfly-GEM/gem-browser/metabolite/MAM03165c

the xref to chebi is correct

https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:16802

but the xref to KEGG is to 
https://www.genome.jp/dbget-bin/www_bget?C02076

and this KEGG has xref to 2 chebi ids:
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:16802 ---> OK
but also
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:27520
which is the pyranose form, and makes things complicated

CHEBI has this information
β-D-sedoheptulopyranose (CHEBI:27520) is a sedoheptulose (CHEBI:16802) 

In KEGG, 2 compounds exist
https://www.genome.jp/entry/C08355

https://www.genome.jp/entry/C02076

but the xref that are present into
https://www.genome.jp/entry/C02076
are maybe wrong, because merging CHEBI:16802 and CHEBI:27520

In summary, I guess, it would be better for MAM03165c not to have xref to KEGG but only to CHEBI:16802

Thoughts?

**Comment by Hao-Chalmers** [comment]
Agree

**Comment by Hao-Chalmers** [comment]
Problem fixed in #312, good job @ANiknejad
